pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "python-sdk"
+version = "0.1.0"
+description = "SDK for integration with the Woovi API"
+authors = ["Camilo Cunha de Azevedo <camilotk@gmail.com>"]
```

Comment:
Add "woovi <developers@woovi.com>

Reply:
Done in [76261f1](https://github.com/Open-Pix/python-sdk/pull/1/commits/76261f1ac0eb42cce1493c894ddba4e2f0d940fe).